src/app/exercise/exercise/exercise.component.ts

```diff
@@ -1,21 +1,24 @@
-import {Component, Input} from '@angular/core';
+import {AfterContentInit, Component, ContentChildren, Input, OnInit, QueryList, ViewChildren} from '@angular/core';
 import {ExerciseBase} from './exercise.base';
 import {FileConfig} from '../interfaces/file-config';
 import {MonacoConfigService} from '../services/monaco-config.service';
 import {SlideComponent} from '../../presentation/slide/slide.component';
 import {AnalyticsService} from '../../presentation/analytics.service';
 import {PresentationComponent} from '../../presentation/presentation/presentation.component';
 import {ActivatedRoute} from '@angular/router';
+import {TranslateTestDescriptionComponent} from '../translate-test-description/translate-test-description.component';
 
 @Component({
   selector: 'slides-exercise',
   templateUrl: 'exercise.component.html',
   styleUrls: ['exercise.component.css']
 })
-export class ExerciseComponent extends ExerciseBase {
+export class ExerciseComponent extends ExerciseBase implements AfterContentInit  {
   @Input() milestone;
   @Input() config;
+  @ContentChildren(TranslateTestDescriptionComponent) testDescriptionComponents: QueryList<TranslateTestDescriptionComponent>;
```

Comment:
Can you use a span or div with a CSS selector? That would eliminate the need for a new empty component. 

```typescript
@ContentChildren('div[description]') descriptionChildren: QueryList<ElementRef>;
```